src/help/changelog.md

```diff
@@ -1,18 +1,13 @@
-#### May 24, 2017
+#### June 9, 2017
 
-On September 30, 2016, we released the full version of the DATA Act Broker that contains everything agencies need to test the data validation and submission process. We continue to make improvements to the Broker and respond to issues discovered through agency use.
+In this release of the Broker, all users are now able to generate D files outside the context of a submission. We also implemented processing time improvements.
 
-In this release of the Broker, we added a certified data column to the submission dashboard, updated the warning/error reports to include the rule label, and updated the help page with information about the Service Desk.
+  - [Generating D files outside of a submission](#/help?section=generatedfile)
+  - [Improved processing time](#/help?section=processingtime)
 
-  - [Certified date column in the submission dashboard](#/help?section=certifieddatecolumn)
-  - [Rule label included in error and warning descriptions](#/help?section=rulelabeldesc)
-  - [Service Desk information on help page](#/help?section=servicedeskinfo)
+##### Generating D files outside of a submission
+{section=generatedfile}
```

Comment:
This should be on the same line as the title itself. I'm not sure if it'll mess anything up if it isn't.